spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/oauth/DataflowOAuthIT.java

```diff
@@ -92,4 +92,60 @@ void securedSetup() throws Exception {
 			}
 		}
 	}
+
+	@Test
+	void securedSetupKeycloak() throws Exception {
+		log.info("Running securedSetupKeycloak()");
+		this.dataflowCluster.startIdentityProvider(TagNames.KEYCLOAK_26);
+		this.dataflowCluster.startSkipper(TagNames.SKIPPER_main);
+		this.dataflowCluster.startDataflow(TagNames.DATAFLOW_main);
+
+		// we can't do oauth flow from host due to how oauth works as we
+		// need proper networking, so use separate tools container to run
+		// curl command as we support basic auth and if we get good response
+		// oauth is working with dataflow and skipper.
+
+		AtomicReference<String> stderr = new AtomicReference<>();
+		try {
+			with().pollInterval(5, TimeUnit.SECONDS)
+					.and()
+					.await()
+					.ignoreExceptions()
+					.atMost(90, TimeUnit.SECONDS)
+					.untilAsserted(() -> {
+						log.info("Checking auth using curl");
+						ExecResult cmdResult = execInToolsContainer("curl", "-v", "-u", "joe:password", "http://dataflow:9393/about");
+						String response = cmdResult.getStdout();
+						if (StringUtils.hasText(response)) {
+							log.info("Response is {}", response);
+						}
+						if(StringUtils.hasText(cmdResult.getStderr())) {
```

Comment:
Again, inconsistent spacing line 122 and 119. please adjust in all newly added code